abtesting/ABTestingExample.xcodeproj/project.pbxproj

```diff
@@ -21,6 +21,13 @@
 		415FC24126D8E90B00AB8835 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 8052DFC0B0123902D171FEAB /* GoogleService-Info.plist */; };
 		415FC24326D8E92200AB8835 /* FirebaseInstallations in Frameworks */ = {isa = PBXBuildFile; productRef = 415FC24226D8E92200AB8835 /* FirebaseInstallations */; };
 		415FC24526D8E92200AB8835 /* FirebaseRemoteConfig in Frameworks */ = {isa = PBXBuildFile; productRef = 415FC24426D8E92200AB8835 /* FirebaseRemoteConfig */; };
+		41689EC126E027DE0053A241 /* ABTestingExampleApp.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4125B7C0268E2E7A00232B67 /* ABTestingExampleApp.swift */; };
+		41689EC226E027DE0053A241 /* ContentView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4125B7C1268E2E7A00232B67 /* ContentView.swift */; };
+		41689EC326E027DE0053A241 /* AppConfig.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4125B84D268E7EF000232B67 /* AppConfig.swift */; };
+		41689EC726E027E80053A241 /* FirebaseRemoteConfig in Frameworks */ = {isa = PBXBuildFile; productRef = 41689EC626E027E80053A241 /* FirebaseRemoteConfig */; };
+		41689EC826E027F30053A241 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 8052DFC0B0123902D171FEAB /* GoogleService-Info.plist */; };
+		41689ECA26E068AA0053A241 /* CoreTelephony.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 41689EC926E068A90053A241 /* CoreTelephony.framework */; };
```

Comment:
CoreTelephony was added explicitly to the app since GoogleDataTransport support for catalyst has not been released yet: https://github.com/google/GoogleDataTransport/pull/36

Reply:
@ncooke3 Can we release an SPM only 9.1.1?

Reply:
Resolved ✅